pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -422,6 +422,21 @@ type ChildStatusReference struct {
 	WhenExpressions []WhenExpression `json:"whenExpressions,omitempty"`
 }
 
+// GetConditionChecksAsMap returns a map representation of this ChildStatusReference's ConditionChecks, in the same form
+// as PipelineRunTaskRunStatus.ConditionChecks.
+func (cr ChildStatusReference) GetConditionChecksAsMap() map[string]*PipelineRunConditionCheckStatus {
```

Comment:
nit: this function can be named "GetConditionChecks" ("asMap" is redundant)

Reply:
👍